From 6f4106f7a7fdcbc03290008713915b4122988c90 Mon Sep 17 00:00:00 2001
From: James Hughes <JamesH65@users.noreply.github.com>
Date: Wed, 5 Jul 2023 15:43:30 +0100
Subject: [PATCH] overlays: audremap: Document CM4 40&41 restriction

Update audremap information to state pins 40,41 are not available on the CM4.

Signed-off-by: James Hughes (james.hughes@raspberrypi.com)
---
 arch/arm/boot/dts/overlays/README | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/arch/arm/boot/dts/overlays/README
+++ b/arch/arm/boot/dts/overlays/README
@@ -703,7 +703,8 @@ Params: swap_lr                 Reverse
                                 nothing on BCM2711 (default off)
         pins_12_13              Select GPIOs 12 & 13 (default)
         pins_18_19              Select GPIOs 18 & 19
-        pins_40_41              Select GPIOs 40 & 41
+        pins_40_41              Select GPIOs 40 & 41 (not available on CM4, used
+                                for other purposes)
         pins_40_45              Select GPIOs 40 & 45 (don't use on BCM2711 - the
                                 pins are on different controllers)
 
